src/radspec.js

```diff
@@ -6,7 +6,7 @@ import { numberToWord } from './lib/math-utils'
 
 export default {
   approveFeeDeposit: amount => {
-    return `Approve fee deposit: ${amount} ANJ`
+    return `Approve fee deposit: ${amount} DAI`
```

Comment:
Would like @fabriziovigevani to triple check, but I think this is correct :).

Reply:
@sohkai to clarify you think my correction is correct, or the original is correct? I created this PR because I was testing and it said to approve ANJ even though the next tx took DAI from my account. And all fees are paid in DAI.

Reply:
Yes, this is correct! It should be `DAI` as you well suggest.